sgkit/__init__.py

```diff
@@ -5,10 +5,12 @@
     DIM_VARIANT,
     create_genotype_call_dataset,
     create_genotype_dosage_dataset,
+    ts_to_dataset,
```

Comment:
I don't think we want this in core library as it's introducing a dependency on tskit and we'd probably want to do it a bit differently anyway. Let's just consider ``ts_to_dataset`` a testing utility for now.